packages/cloud-cognitive/src/components/CreateSidePanel/CreateSidePanel.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Import portions of React that are needed.
+import React from 'react';
+
+// Other standard imports.
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { pkg } from '../../settings';
+
+// Carbon and package components we use.
+import { Form } from 'carbon-components-react';
+import { SidePanel } from '../SidePanel/SidePanel';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--create-side-panel`;
+const componentName = 'CreateSidePanel';
+
+// NOTE: the component SCSS is not imported here: it is rolled up separately.
+
+/**
+ * This is an example component to show relevant conventions and usage.
+ */
+export let CreateSidePanel = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+      className,
+      onRequestClose,
+      open,
+      title,
+      subtitle,
+      children,
+      disableSubmit,
+      onRequestSubmit,
+      primaryButtonText,
+      secondaryButtonText,
+      selectorPrimaryFocus,
+      pageContentSelector,
+      formTitle,
+      formDescription,
+      ...rest
+    },
+    ref
+  ) => {
+    const actions = [
+      {
+        label: primaryButtonText,
+        onClick: () => {
+          onRequestSubmit();
+        },
```

Comment:
this can be simplified to `onClick: onRequestSubmit,`

Reply:
Good catch.